python/cucim/src/cucim/skimage/feature/tests/test_corner.py

```diff
@@ -342,7 +342,7 @@ def _reference_eigvals_computation(S_elems):
 def test_custom_eigvals_kernels_vs_linalg_eigvalsh(shape, dtype):
     rng = cp.random.default_rng(seed=5)
     img = rng.integers(0, 256, shape)
-    H = hessian_matrix(img)
+    H = hessian_matrix(img, use_gaussian_derivatives=False)
```

Comment:
Will we need to change [the `hessian_matrix`'s use `use_gaussian_derivatives` default value]( https://github.com/rapidsai/cucim/blob/6d5b8741ee26e04716653f22602682010f7058df/python/cucim/src/cucim/skimage/feature/corner.py#L252 ) at some point?

Reply:
Not sure. It is not marked for change yet in upstream `scikit-image`.

Reply:
Maybe we can write up an issue to track